addons/mail/static/src/components/attachment_viewer/attachment_viewer.scss

```diff
@@ -147,6 +152,20 @@
         background-color: rgba(0, 0, 0, 0.8);
         color: lighten(gray('400'), 15%);
     }
+    &.btn-primary {
+        @include button-variant($o-brand-primary, $o-brand-primary);
+    }
+    @include media-breakpoint-down(sm) {
+        &.btn {
+            padding: 0;
+            border: 0;
+            background: none;
+
+            span {
+                display: none;
+            }
```

Comment:
`display: none` should never be used. Node should conditionally not be rendered in the template if it's not needed (can use `env.messaging.device.isMobile` to make the condition).